lib/pyfrc/physics/drivetrains.py

```diff
@@ -176,7 +176,7 @@ def four_motor_swerve_drivetrain(lr_motor, rr_motor, lf_motor, rf_motor, lr_angl
     rf_rad = rf_angle * (math.pi / 180)
 
     # Calculate wheelbase radius
-    wheelbase_radius = math.sqrt((x_wheelbase ** 2) + (y_wheelbase ** 2))
+    wheelbase_radius = math.sqrt(((x_wheelbase/2) ** 2) + ((y_wheelbase/2) ** 2))
```

Comment:
I'd personally argue that having superfluous parentheses when order of operations is obvious, such as in this scenario, hurts readability. I had to keep glancing left and right to work out what the order of operations is, when it's just normal maths here.